docs/bokeh/source/docs/user_guide/basic/axes.rst

```diff
@@ -90,11 +90,6 @@ For information on how to set your own ranges, see
 .. bokeh-plot:: __REPO__/examples/basic/axes/log_scale_axis.py
     :source-position: above
 
-Mercator axes
-~~~~~~~~~~~~~
-
-.. TODO (bv) comments
-
```

Comment:
Rather than removing this, we could add a minimal sentence about how mercator axes are useful with tile sources, setting the axis type values to "mercator" and than add this example: `__REPO__/examples/topics/geo/tile_source.py` similar to the section just above. @bryanwweber would you be able to do that?

Reply:
Happy to do so! I'll try to get to it this weekend, or latest early next week. Thanks for the quick review!